packages/react-components/src/accordion/stories/accordion.stories.tsx

```diff
@@ -7,12 +7,16 @@ export default {
   component: Accordion,
   title: 'Data Display/Accordion',
   args: {
-    type: 'single'
+    type: 'single',
+    itemDisabled: false
   },
   argTypes: {
     type: {
       control: 'radio',
       options: ['single', 'multiple']
+    },
+    itemDisabled: {
```

Comment:
@tmbalagan  Can we move this to docs (`packages/docs/docs/components/accordion.mdx`) ? We can add the `disabled` prop to the `AccordionItem` prop table which we already have. Calling it `itemDisabled` here while the actual prop is `disabled` can be confusing.

Reply:
@maneeshp97 updated, Kindly take a look.